$NetBSD: patch-src_glx_Makefile.am,v 1.1 2019/01/19 21:54:03 tnn Exp $

- FDO bug 90311
- We don't need libmesautil.la here. Those symbols seem to already be
  included in libmesa.la.

--- src/glx/Makefile.am.orig	2019-01-17 11:26:23.000000000 +0000
+++ src/glx/Makefile.am
@@ -89,10 +89,16 @@ libglx_la_SOURCES = \
 	singlepix.c \
 	vertarr.c
 
+if HAVE_APPLEDRI
+libglx_la_LIBADD = \
+	$(top_builddir)/src/loader/libloader.la \
+	$(top_builddir)/src/util/libxmlconfig.la
+else
 libglx_la_LIBADD = \
 	$(top_builddir)/src/loader/libloader.la \
 	$(top_builddir)/src/util/libmesautil.la \
 	$(top_builddir)/src/util/libxmlconfig.la
+endif
 
 if HAVE_DRISW
 libglx_la_SOURCES += \
@@ -136,6 +142,8 @@ libglx_la_SOURCES += \
 
 SUBDIRS += apple
 libglx_la_LIBADD += $(builddir)/apple/libappleglx.la
+libglx_la_LIBADD += $(top_builddir)/src/mesa/libmesa.la
+libglx_la_SOURCES += dummy.cpp
 endif
 
 if HAVE_WINDOWSDRI
